src/Main.css

```diff
@@ -20,7 +20,7 @@
 }
 
 * {
```

Comment:
Seems there is already a CSS for * font-family as "Lato", sans-serif. Shouldn't u be editing that instead of creating a new style for the same.